third-party/zen-pmu-workaround/README

```diff
@@ -0,0 +1,2 @@
+Mike Hommey's Zen workaround kernel module.
```

Comment:
Maybe mention that it does the same thing as scripts/zen_workaround.py. and/or zen_workaround.service in a way that persists after resume from sleep. (BTW, now that I discovered there's a systemd service file, there's probably a way to restore zen_workaround.py on resume with it too)

I'd also start with Zen workaround kernel module, and put something like "Copyright 2020, Mike Hommey" (as to indicate the age of this thing)

Not sure this warrants a separate file though.

Reply:
I think the kernel module is also the only way to do it when you have SecureBoot enabled, python script was failing for me with that setup.